md/minimizer.py

```diff
@@ -70,15 +81,21 @@ def minimize_host_4d(romol, host_system, host_coords, ff, box):
         params, potential = fn(*[h.params for h in handles])
         final_potentials.append(potential.bind(params))
 
-    seed = 2020
+    # this value doesn't matter since we will turn off the noise.
+    seed = 0
 
     intg = LangevinIntegrator(
         300.0,
         1.5e-3,
         1.0,
         combined_masses,
         seed
-    ).impl()
+    )
+
+    # remove the randomness completely
+    intg.ccs = np.zeros_like(intg.ccs)
```

Comment:
ccs isn't the friction coefficient, but its derived from it the cas, cbs, ccs are computed via:

``` python
# timemachine/integrator.py

def langevin_coefficients(
    temperature,
    dt,
    friction,
    masses):
    """
    Compute coefficients for langevin dynamics

    Parameters
    ----------
    temperature: float
        units of Kelvin

    dt: float
        units of picoseconds

    friction: float
        frequency in picoseconds

    masses: array
        mass of each atom in standard mass units

    Returns
    -------
    tuple (ca, cb, cc)
        ca is scalar, and cb and cc are n length arrays
        that are used during langevin dynamics

    """
    vscale = np.exp(-dt*friction)
    if friction == 0:
        fscale = dt
    else:
        fscale = (1-vscale)/friction
    kT = BOLTZ * temperature
    nscale = np.sqrt(kT*(1-vscale*vscale)) # noise scale
    invMasses = 1.0/masses
    sqrtInvMasses = np.sqrt(invMasses)

    ca = vscale
    cb = fscale*invMasses
    cc = nscale*sqrtInvMasses
    return ca, cb, cc
```

So there's still drag

Reply:
Ahh, thanks for clarifying! I had mis-remembered this. Marking resolved

Reply:
It may still be clearer to set the temperature to 0 in the integrator constructor, rather than overriding these coefficients, but the effect should be the same

Reply:
Good call! 
```
>>> integrator.langevin_coefficients(300.0, 1.5e-3, 1.0, np.array([1.0]))
(0.9985011244377109, array([0.00149888]), array([0.08643984])) # ca cb cc
>>> integrator.langevin_coefficients(0.0, 1.5e-3, 1.0, np.array([1.0]))
(0.9985011244377109, array([0.00149888]), array([0.])) # ca cb cc
```

I like this better